.github/workflows/test.yaml

```diff
@@ -87,7 +87,23 @@ jobs:
         with:
           tool: cargo-make
 
+      - name: Check if Release PR
+        id: check_release_pr
+        shell: bash
+        run: |
+          # github.head_ref is only available for pull_request events
+          # For push events, it will be empty, so we default to running all tests
+          if [[ "${{ github.head_ref }}" == release-plz-* ]]; then
+            echo "skip_driver_build=1" >> $GITHUB_OUTPUT
+            echo "This is a release-plz PR - will skip driver build in tests"
```

Comment:
```suggestion
            echo "This is a release-plz PR. Will skip the build part in cargo-wdk new integration tests because the needed deps may not yet have been published"
```